.github/workflows/merge.yml

```diff
@@ -0,0 +1,24 @@
+name: merge
+
+# open or update publishing PR when there is a push to main
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  main-to-published:
+    runs-on: ubuntu-22.04
+    if: github.repository_owner == 'docker'
+    steps:
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          delete-branch: false
```

Comment:
I think the branch should be removed when merged

Reply:
I think it opens directly from the `main` branch, so ... probably not? 😅

Reply:
Oh boy right it does not fork :see_no_evil: